nevergrad/functions/multiobjective/hypervolume.py

```diff
@@ -1,6 +1,9 @@
 # (C) Copyright 2020 Enthought, Inc., Austin, TX
```

Comment:
I did not realize this file did not have the Facebook header either
Can you also add `# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.` on top? We need to enforce this on all files, I thought that had been added in the initial PR. As far as I understand, you can leave the Enthought copyright along as well.

Reply:
Please also add it to `test_structures.py` btw

Reply:
Thank you, @jrapin . This should be fixed now.